Source/ACE.Server/WorldObjects/Player.cs

```diff
@@ -58,8 +59,6 @@ public Player(Weenie weenie, ObjectGuid guid, uint accountId) : base(weenie, gui
             Attackable = true;
 
             SetProperty(PropertyString.DateOfBirth, $"{DateTime.UtcNow:dd MMMM yyyy}");
-
-            SetEphemeralValues();
```

Comment:
Why the removal of this call? SetEphemeralValues is a pretty critical call to the setting up of the object.

Reply:
I did some further checking and this is what I found.
Existing characters and Character creation both call...
ACE.Server\Managers\WorldManager.cs -> DoPlayerEnterWorld
that calls ACE.Server\WorldObjects\Player.cs -> Player which calls SetEphemeralValues

On new characters with the code in this pull request. 
ACE.Server\Network\Handlers\CharacterHandler.cs
Now calls. ACE.Server\Managers\WorldManager.cs -> AddPlayer
that calls ACE.Server\WorldObjects\Player.cs -> Player which calls SetEphemeralValues

So in the end we are still calling it twice on character creation even with the first call removed.

Reply:
SetEphemeralValues needs to be called from every ctor. If you remove it from Player, it should be removed from every child/parent from weenie ctor, which isn't right.

If there's an issue you're trying to solve by removing the call, then the problem is likely that there is code in SetEphemeralValues that shouldn't be in there.

Also, Character.IsPlussed = false; shouldn't be there as well. No need to set it to the default value.

Feel free to reach out to me in discord.